src/login/login.service.ts

```diff
@@ -1,4 +1,30 @@
-import { Injectable } from '@nestjs/common';
+import { Injectable, HttpException, HttpStatus } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { UserRepository } from './user.repository';
+import { AuthCredentialDto } from './dto/auth-credential.dto';
+import { compare } from 'bcrypt';
 
 @Injectable()
-export class LoginService {}
+export class LoginService {
+  constructor(
+    @InjectRepository(UserRepository)
+    private readonly userRepository: UserRepository,
```

Comment:
직접 레포지토리를 만들지 않아도 typeorm에서 만들어주는 코드가 있어요.
```suggestion
    @InjectRepository(User)
    private readonly userRepository: Repository<Checkin>,
```

esports backend에서는 이렇게 사용하였어요. 만약 이렇게 사용한다면 repository 클래스를 직접 만들지 않으셔도 되어요.